pyproject.toml

```diff
@@ -0,0 +1,136 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "clickhouse-sqlalchemy"
+version = "0.3.2"
+description = "SQLAlchemy dialect for ClickHouse"
+authors = [
+    {name = "Konstantin Lebedev", email = "kostyan.lebedev@gmail.com"},
+]
+readme = "README.md"
+license = {text = "MIT"}
+requires-python = ">=3.7"
+keywords = ["clickhouse", "sqlalchemy", "orm", "dialect"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: SQL",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Application Frameworks",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "asynch>=0.2.2,!=0.2.4,!=0.2.5,<0.4.0",
+    "clickhouse-driver>=0.2.9",
+    "requests>=2.31.0",
+    "sqlalchemy>=1.4",
+]
+
+[project.urls]
+Homepage = "https://github.com/xzkostyan/clickhouse-sqlalchemy"
+Documentation = "https://clickhouse-sqlalchemy.readthedocs.io/en/latest/"
+Repository = "https://github.com/xzkostyan/clickhouse-sqlalchemy"
+Issues = "https://github.com/fastapi/fastapi/issues"
+Changelog = "https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/CHANGELOG.md"
+PyPI = "https://pypi.org/project/clickhouse-sqlalchemy/"
+
+[project.entry-points."sqlalchemy.dialects"]
+"clickhouse" = "clickhouse_sqlalchemy.drivers.http.base:ClickHouseDialect_http"
+"clickhouse.asynch" = "clickhouse_sqlalchemy.drivers.asynch.base:ClickHouseDialect_asynch"
+"clickhouse.http" = "clickhouse_sqlalchemy.drivers.http.base:ClickHouseDialect_http"
+"clickhouse.native" = "clickhouse_sqlalchemy.drivers.native.base:ClickHouseDialect_native"
```

Comment:
Ahem, could you give a hint how to test the fact these entry points work as before?